runtime/default.policy

```diff
@@ -62,132 +94,153 @@ rules:
-      && ((open.filename !~ "/var/log/datadog/*" && process.filename not in DD_AGENT_PROCESSES) || process.filename not in CONTAINER_PROCESSES)
+      (open.flags&OPEN_WRITE_FLAGS > 0) &&
+      (open.file.path in [~"/var/log/pods/*", ~"/var/log/containers/*"] && process.file.path not in KUBELET_PROCESSES) ||
+      (open.file.path =~ "/var/log/journal/*" && process.file.path not in SYSTEMD_JOURNALD_PROCESSES) ||
+      (open.file.path =~ "/var/log/fondationdb/*" && process.file.path not in FDB_SERVER_PROCESSES) ||
+      (open.file.path =~ "/var/log/apt/*" && process.file.path not in APT_PROCESSES) ||
+      (open.file.path =~ "/var/log/datadog/*" && process.file.path not in DD_AGENT_PROCESSES) ||
+      (open.file.path =~ "/var/log/*" &&
+       process.file.path not in KUBELET_PROCESSES &&
+       process.file.path not in SYSTEMD_JOURNALD_PROCESSES &&
+       process.file.path not in FDB_SERVER_PROCESSES &&
+       process.file.path not in APT_PROCESSES &&
+       process.file.path not in DD_AGENT_PROCESSES)
 
   - id: logs_removed
-    description: Log files were removed
+    description: Log data was deleted
     expression: >-
-      unlink.filename =~ "/var/log/*"
-      && ((unlink.filename !~ "/var/log/datadog/*" && process.filename not in DD_AGENT_PROCESSES) || process.filename not in CONTAINER_PROCESSES)
+      (unlink.file.path in [~"/var/log/pods/*", ~"/var/log/containers/*"] && process.file.path not in KUBELET_PROCESSES) ||
+      (unlink.file.path =~ "/var/log/journal/*" && process.file.path not in SYSTEMD_JOURNALD_PROCESSES) ||
+      (unlink.file.path =~ "/var/log/fondationdb/*" && process.file.path not in FDB_SERVER_PROCESSES) ||
+      (unlink.file.path =~ "/var/log/apt/*" && process.file.path not in APT_PROCESSES) ||
+      (unlink.file.path =~ "/var/log/datadog/*" && process.file.path not in DD_AGENT_PROCESSES) ||
+      (unlink.file.path =~ "/var/log/*" &&
+       process.file.path not in KUBELET_PROCESSES &&
+       process.file.path not in SYSTEMD_JOURNALD_PROCESSES &&
+       process.file.path not in FDB_SERVER_PROCESSES &&
+       process.file.path not in APT_PROCESSES &&
+       process.file.path not in DD_AGENT_PROCESSES)
 
   - id: permissions_changed
     description: Permissions were changed on sensitive files
     expression: >-
-      (chmod.filename =~ "/etc/*" ||
-      chmod.filename =~ "/sbin/*" || chmod.filename =~ "/usr/sbin/*" ||
-      chmod.filename =~ "/usr/local/sbin/*" || chmod.filename =~ "/usr/local/bin/*" ||
-      chmod.filename =~ "/var/log/*" || chmod.filename =~ "/usr/lib/*") &&
-      process.filename not in CONTAINER_PROCESSES
+      (chmod.file.path =~ "/etc/*" ||
+      chmod.file.path =~ "/sbin/*" || chmod.file.path =~ "/usr/sbin/*" ||
+      chmod.file.path =~ "/usr/local/sbin/*" || chmod.file.path =~ "/usr/local/bin/*" ||
+      chmod.file.path =~ "/var/log/*" || chmod.file.path =~ "/usr/lib/*") &&
+      process.file.path not in CONTAINER_PROCESSES
```

Comment:
Should we allow-list package managers here? We do it for the creation of new binaries, so this feels like it follows a similar pattern.

Reply:
Agreed